helmfile/50-applications.yaml

```
@@ -391,6 +406,23 @@ releases:
 
 # Workload cluster releases
 {{ if eq .Environment.Name "workload_cluster" }}
+
+# prometheus wc-servicemonitor
+- name: wc-servicemonitor
+  namespace: monitoring
+  labels:
+    app: prometheus-servicemonitor
+    prometheus: wc-reader
+  chart: ./charts/prometheus-servicemonitor
+  version: 0.1.1
+  missingFileHandler: Error
+  values:
+{{ if eq .Environment.Name "workload_cluster" }}
+  # Note: we want to have alerts for WC since this is wc-reader, even if it is
+  # running in SC.
+  - values/wc-reader-servicemonitor.yaml.gotmpl
+{{ end }}
```

You have this inside a similar if-clause already.
```suggestion
  - values/wc-reader-servicemonitor.yaml.gotmpl
```